app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Eav.php

```diff
@@ -614,6 +624,149 @@ private function isProductExists()
+                        foreach ($stores as $storeId) {
+                            if ($this->scopeOverriddenValue->containsValue(
+                                ProductInterface::class,
+                                $this->locator->getProduct(),
+                                $attribute->getAttributeCode(),
+                                $storeId
+                            )) {
+                                $overridden[] = [
+                                    'scope' => 'store',
+                                    'scope_id' => $storeId,
+                                ];
+                            }
+                        }
+                    }
+                    break;
+                default:
+                    if (isset($tree[ProductAttributeInterface::SCOPE_WEBSITE_TEXT])) {
+                        foreach ($tree[ProductAttributeInterface::SCOPE_WEBSITE_TEXT] as $website) {
+                            foreach ($website[ProductAttributeInterface::SCOPE_STORE_TEXT] as $storeId) {
+                                if ($this->scopeOverriddenValue->containsValue(
+                                    ProductInterface::class,
+                                    $this->locator->getProduct(),
+                                    $attribute->getAttributeCode(),
+                                    $storeId
+                                )) {
+                                    $overridden[] = [
+                                        'scope' => 'store',
+                                        'scope_id' => $storeId,
+                                    ];
+                                }
+                            }
+                        }
+                    }
+                    break;
+            }
+        }
+
+        return $overridden;
+    }
+
+    /**
+     * @return array
+     */
+    private function getScopeTree()
+    {
+        $tree = [ProductAttributeInterface::SCOPE_WEBSITE_TEXT => []];
+
+        $websites = $this->storeManager->getWebsites(true);
+        if (!$websites) {
+            return false;
+        }
+
+        /* @var $website Website */
+        foreach ($websites as $website) {
+            $tree[ProductAttributeInterface::SCOPE_WEBSITE_TEXT][$website->getId()] = [ProductAttributeInterface::SCOPE_STORE_TEXT => []];
+            /* @var $store Store */
+            foreach ($website->getStores() as $store) {
+                $tree[ProductAttributeInterface::SCOPE_WEBSITE_TEXT][$website->getId()][ProductAttributeInterface::SCOPE_STORE_TEXT][] = $store->getId();
+            }
+        }
+
+        return $tree;
+    }
+
+    /**
+     * @param array $overridden
+     * @return string
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    private function formatOverriddenScopes(array $overridden)
+    {
+        $formatted = '<div class="overridden-hint-wrapper">' .
```

Comment:
Try to render HTML in the view layer (html file for the field). In the modifier, we are allowed to return only raw data